net/pfSense-pkg-WireGuard/Makefile

```diff
@@ -0,0 +1,79 @@
+# $FreeBSD$
+
+PORTNAME=	pfSense-pkg-WireGuard
+PORTVERSION=	0.0.6
+PORREVISION=	3
+CATEGORIES=	net
+MASTER_SITES=	# empty
+DISTFILES=	# empty
+EXTRACT_ONLY=	# empty
+
+
+MAINTAINER=	rcmcdonald91@gmail.com
+COMMENT=	pfSense package WireGuard
+
+LICENSE=	APACHE20
+
+RUN_DEPENDS=	${LOCALBASE}/bin/bash:shells/bash \
+		wg:net/wireguard-tools \
+		wg-quick:net/wireguard-tools \
+		${KMODDIR}/if_wg.ko:net/wireguard-kmod
```

Comment:
Use path to detect a port is the old way, today use `portname>=0:category/port-directory` is preferred, as follow:

```
RUN_DEPENDS=	bash>=0:shells/bash \
		wireguard-tools>0:net/wireguard-tools \
		wsireguard-kmod>0:net/wireguard-kmod
```

Also remove duplicated wireguard-tools entry